mitiq/calibration/tests/test_settings.py

```diff
@@ -304,3 +305,12 @@ def test_to_dict():
         "factory": "LinearFactory",
         "scale_factors": [1.0, 2.0],
     }
+
+
+def test_num_circuits_required():
+    undefine_strategy = Strategy(
+        id=1,
+        technique=MitigationTechnique.RAW,
+        technique_params={},
+    )
+    assert undefine_strategy.num_circuits_required() == 0
```

Comment:
Good thinking with this test. It made me realize that we should actually have another case in `num_circuits_required` where `RAW` should return 1 since raw execution just means execute the circuit.

If you add that as a case in `num_circuits_required` you can change this test so that it passes, and please ensure to use a specific name for this test, for example `test_num_circuits_required_raw_execution`. If you do this, you'll have to add another test (maybe `test_num_circuits_unknown_method`) to assert num circuits is 0.

Sorry for the back and forth! Once this change is made, it's ready to go!